readme.md

```diff
@@ -2,6 +2,8 @@
 
 [![Build Status](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Factions-badge.atrox.dev%2Fblockstack%2Fstacks-blockchain-sidecar%2Fbadge%3Fref%3Dmaster&style=flat)](https://actions-badge.atrox.dev/blockstack/stacks-blockchain-sidecar/goto?ref=master)
 
+[![Gitpod ready-to-code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/blockstack/stacks-blockchain-sidecar)
```

Comment:
I'm also getting an error running the faucet. Is there some hardcoded base URL in our codebase?

<img width="1096" alt="Screen Shot 2020-07-24 at 8 45 42 AM" src="https://user-images.githubusercontent.com/1671228/88409695-29e0ca00-cd8a-11ea-840c-0755fda36b9a.png">

Reply:
Fixed